fjord/alerts/admin.py

```diff
@@ -5,7 +5,7 @@
 
 class AlertFlavorAdmin(admin.ModelAdmin):
     list_display = (
-        'enabled', 'name', 'slug', 'description', 'more_info',
+        'name', 'slug', 'enabled', 'description', 'more_info',
```

Comment:
Changing this means that "name" is the left-most column in the admin table and is the one that you click on for details. That's easier to use than "enabled".